### DISCUSSION TITLE
Use next/image in pagination

### DISCUSSION THREAD

**Discussion by comoestassergio** [nolimits4web/swiper#6250] [Q&A] [answered]
Hey there! 

I'm trying to use **next/image** inside my custom pagination and haven't been able to do that so far. The **renderBullet function,** as it seems, just **ignores custom components** and renders everything with _plain html_. So, my Image tag falls back to plain img in the end.

Is there a way to implement it?

![image](https://user-images.githubusercontent.com/93369654/205239112-a4c18747-bedd-4434-96c9-ac337f9f3e54.png)

**Comment by onezeno717** [comment, marked as answer]
why not use Thumbnail? https://swiperjs.com/demos#thumbs-gallery

**Reply by comoestassergio** [reply]
Yeah, thanks! 
That's actually the solution I ended up with.
